src/DotNetCore.CAP.RabbitMQ/IConnectionChannelPool.Default.cs

```diff
@@ -98,7 +98,10 @@ public virtual IModel Rent()
 
                 Debug.Assert(_count >= 0);
 
-                return model;
+                if (model.IsOpen && model.CloseReason == null)
```

Comment:
If we have already detected the state of the connection at the time of the return, I think it is unnecessary to check again

Reply:
The connection can be disconnected at any time even it has been returned. It should better add a safe check here.